### ISSUE TITLE
Markdown code block should have a background

### ISSUE THREAD

**Issue by olivierdalang** [Microsoft/vscode#41112]
Markdown code highlighting doesn't make code blocks stand out, making it hard to read a markdown document without the preview.

- VSCode Version: 1.19.1
- OS Version: Windosw 10

Steps to Reproduce:

1. Create a markdown file (.md)
2. Create a multiline code block with ```
3. Notice that the block doesn't get any type of rendering, making it hard to follow the document's structure

Here's how it looks like with current VSCode :

![capture](https://user-images.githubusercontent.com/1894106/34538942-59321d3a-f133-11e7-90b8-d92b0761f6de.PNG)

Here's an example rendering where code blocks have a different background, much better ! (from sublime with markdown extended package)

![capture2](https://user-images.githubusercontent.com/1894106/34538941-58fa362c-f133-11e7-91f4-955aa824c761.PNG)

**Comment by mjbvz** [comment]
This would be something a theme could control. Currently blocked by #3429

Closing as duplicate of #3429 + future extension candidate
